tsconfig.json

```diff
@@ -64,6 +64,9 @@
     {
       "path": "./packages/preferences-controller"
     },
+    {
+      "path": "./packages/queued-request-controller"
+    },
```

Comment:
I believe this is here already. It was just added to the bottom.

``` suggestion
```

I also checked `tsconfig.build.json` and it's there too (also at the bottom)